frontend/tools/rectangle.ts

```diff
@@ -38,6 +39,15 @@ class Rectangle {
       stopClick: true,
       style: this.styles,
       geometryFunction: createBox(),
+      finishCondition: () => {
```

Comment:
i like the idea of having a validate function on the tools itself, instead of cramed in the sidebar.

currently this is not called, and I believe its probably because i made it hard to get to the function.. i think we should maybe rename it to validateInput or something similar instead of finishCondtion?

Reply:
i'm not at my pc over the christmas days and really would like to test it out how it feels. i could imagine that this could also a bit confusing, but if you think its better this way, i believe you. just want to test it before releasing it to public.